README.md

```diff
@@ -67,10 +67,16 @@ Navigate to the [chrome://extensions](chrome://extensions) page, enable
 
 ### Firefox
 
+For Firefox >48: Build the extensions with `WITH_ADDON_ID=true yarn build`
+
 If you just want to try out and debug the extension, go to
 [about:debugging#addons](about:debugging#addons). Then press "Load Temporary
 Add-On" and select the `manifest.json` from the built extension directory.
 
+If you want to install this addon permanently goto [about:addons](about:addons)
+and klick on the small cogs icon. Select `Install Addon from file...` and choose
```

Comment:
A few smaller typos:

1. goto => go to
2. klick => click
3. cogs => cog
4. Install Addon from file... => Install Add-on From File...

Reply:
Me != English